### DISCUSSION TITLE
Swup initialized twice on a specific page

### DISCUSSION THREAD

**Discussion by andresclua** [swup/swup#650] [Help] [answered]
I'm kind of new to swup.js
I'm having a specific page that is firing twice, and the rest of my pages are only firing once.
Any ideas?
**About - issue**
![Captura de pantalla 2023-05-15 a las 23 38 55](https://github.com/swup/swup/assets/612806/7fe14305-d0c8-4454-a745-e38dae19b9d5)
**Home**
![Captura de pantalla 2023-05-15 a las 23 40 56](https://github.com/swup/swup/assets/612806/cfecdbb0-4ea6-4f75-af63-99a70473e785)

**Comment by daun** [comment, marked as answer]
The `enabled` event firing again would suggest you're re-initializing swup somewhere after it's already initialized. Make sure you're only setting up swup once when the actual page loads.
